### PR TITLE
Integration test updated for multi LDAP

### PR DESCRIPTION
**Pull request by Christoffer-Cortes**
### Applicable Issues
https://github.com/eiffel-community/eiffel-intelligence/pull/315

### Description of the Change
Tests mirror the ones in back-end function tests. Docker compose needs to start two ldap servers now for the tests to work.

### Sign-off
Developer's Certificate of Origin 1.1

By making a contribution to this project, I certify that:

(a) The contribution was created in whole or in part by me and I
    have the right to submit it under the open source license
    indicated in the file; or

(b) The contribution is based upon previous work that, to the best
    of my knowledge, is covered under an appropriate open source
    license and I have the right under that license to submit that
    work with modifications, whether created in whole or in part
    by me, under the same open source license (unless I am
    permitted to submit under a different license), as indicated
    in the file; or

(c) The contribution was provided directly to me by some other
    person who certified (a), (b) or (c) and I have not modified
    it.

(d) I understand and agree that this project and the contribution
    are public and that a record of the contribution (including all
    personal information I submit with it, including my sign-off) is
    maintained indefinitely and may be redistributed consistent with
    this project or the open source license(s) involved.

Signed-off-by: Christoffer Cortes Sjöwall, christoffer.cortes.sjowall@ericsson.com

### CODE REVIEW

File: src/main/docker/env.bash [hunk 1]
```diff
@@ -16,6 +16,7 @@ export REMREM_PUBLISH_IMAGE="eiffelericsson/eiffel-remrem-publish:2.0.2"
 export JENKINS_IMAGE="bitnami/jenkins:2.138.3"
 export EI_BACKEND_IMAGE="eiffelericsson/eiffel-intelligence-backend:1.0.2"
 export EI_FRONTEND_IMAGE="eiffelericsson/eiffel-intelligence-frontend:1.0.3"
+export LDAP_IMAGE="osixia/openldap"
```
Comment (tobiasake) [inline src/main/docker/env.bash]: Can you set a specific version, so we don't use latest image and risk of getting an new version of openldap that breaks our tests. E.g. "osixia/openldap:1.2.4"

Comment (Christoffer-Cortes) [reply src/main/docker/env.bash]: Done.

File: src/main/docker/docker-compose.yml [hunk 2]
```diff
@@ -437,13 +435,35 @@ services:
...
+    image: ${LDAP_IMAGE}
     volumes:
       - ./ldap-seed:/container/service/slapd/assets/test/
     links:
       - ldap
-    command: chmod +x /container/service/slapd/assets/test/addAll.sh
-    entrypoint: sh -c '/container/service/slapd/assets/test/addAll.sh'
+    entrypoint: bash -c 'sleep 5s && ldapadd -h ldap -D "cn=admin,dc=example,dc=org" -w admin -f /container/service/slapd/assets/test/ldap-users-first.ldif'
+
+  ldap2:
```
Comment (e-pettersson-ericsson) [inline src/main/docker/docker-compose.yml]: In order to test this in Travis, shouldn't these containers also be started in Travis integrationTest job then?

Comment (Christoffer-Cortes) [reply src/main/docker/docker-compose.yml]: I had missed this little detail... Done!

**Pull request merged**